modules.d/01systemd-udevd/module-setup.sh

```diff
@@ -71,6 +71,8 @@ install() {
 
     # Install required libraries.
     _arch=${DRACUT_ARCH:-$(uname -m)}
-    inst_libdir_file {"tls/$_arch/",tls/,"$_arch/",}"libudev.so.*"
+    inst_libdir_file \
+        {"tls/$_arch/",tls/,"$_arch/",}"libudev.so.*" \
+        {"tls/$_arch/",tls/,"$_arch/",}"libkmod.so*"
```

Comment:
I think a better solution would be to remove the change from systemd-udevd and unconditionally install libkmod.so* in udev-rules as systemd-udev has a dependency on udev-rules. Would you be able to make the change and retest please ?

Reply:
Indeed. For some reason I thought that systemd-udevd is also trying to install udev binary, but it's just checking its existence. So it's logical to install udev deps from udev-rules module.